packages/platforms/browser/lib/auto-instrumentation/route-change-plugin.ts

```diff
@@ -20,14 +19,25 @@ export class RouteChangePlugin implements Plugin<BrowserConfiguration> {
         startTime: options ? options.startTime : undefined
       })
 
+      const url = getAbsoluteUrl(route, this.document.baseURI)
```

Comment:
@gingerbenw `route` is an arbitrary string, not necessarily a URL or path

e.g. it's valid to have a route like `user #1234` instead of `/users/1234`

Reply:
good point! hence providing the route resolution function! I shall raise a new PR